src/main/java/randoop/util/ClassFileConstants.java

```diff
@@ -393,9 +446,103 @@ public static ConstantSet getConstants(String classname, ConstantSet result) {
               // Push a value from the constant pool. We'll get these
               // values when processing the constant pool itself.
             case Const.LDC:
+              {
+                LDC ldcInstruction = (LDC) inst;
+                int index = ldcInstruction.getIndex();
+                Constant constant = constant_pool.getConstant(index);
+
+                if (constant instanceof ConstantString) {
+                  String bytes = ((ConstantString) constant).getBytes(constant_pool);
+                  result.constantFrequency.put(
+                      bytes, result.constantFrequency.getOrDefault(bytes, 0) + 1);
```

Comment:
If you want, here and elsewhere you can use
[`CollectionsPlume.incrementMap(map, key)`](https://plumelib.org/plume-util/api/org/plumelib/util/CollectionsPlume.html#incrementMap(java.util.Map,K)).